README.md

```diff
@@ -189,10 +239,9 @@ brightcovePlayerLoader({
 ```
 
 #### `catalogSequence`
-* *Type:* Array | Object
-* *Default:* `undefined`
+* *Type:* `Array` | `Object`
 
-**NOTE:** Only supported for Brightcove Player versions 6.18.0 and newer.
+> **NOTE:** Only supported for Brightcove Player versions 6.18.0 and newer.
 
 A Video Cloud Catalog sequence to perform. See [the Catalog `getSequence` method documentation][bc-sequence] for more.
```

Comment:
This will actually use `getLazySequence`, we don't have a way to call `getSequence` from the config/embed.